webroot/src/components/Page/PageHeader/PageHeader.vue

```diff
@@ -7,7 +7,13 @@
 
 <template>
     <div class="page-header">
-        <div class="header-content"></div>
+        <div class="header-content">
+            <CompactSelector
+                v-if="isStaffUser"
+                :isPermissionBased="true"
+                :hideIfNotMultiple="true"
```

Comment:
nit but any particular design reason to not just include this check in the `v-if` expression on line 12? Seems like it would simplify the code since this is only used in one place at the moment (unless you expect this logic to be repeated?)

Reply:
I suspect it will be repeated (the ability to switch compacts) - such as in the upcoming public UI, or even some super-admin features we've been discussing. It can act as a simple compact dropdown in a broader form, and can also be configured to meet more specific cases like, this this one.

Reply:
Got it thanks!